lib/services/android-project-service.ts

```diff
@@ -227,8 +228,8 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 
 	private getProjectNameFromId(projectData: IProjectData): string {
 		let id: string;
-		if (projectData && projectData.projectId) {
-			const idParts = projectData.projectId.split(".");
+		if (projectData && projectData.projectIdentifiers && projectData.projectIdentifiers.android) {
+			const idParts = projectData.projectIdentifiers.android.split(".");
```

Comment:
split by "." would imply we're sure there is a ".", is there a validation of the package identifier upstream?